### PR TITLE
Fix youcanhelp

### PR DESCRIPTION
**Pull request by gizemmdeniz**
Desktop
image

Ipad
image

Mobile
image

### CODE REVIEW

File: src/components/YouCanHelp/index.jsx [hunk 1]
```diff
@@ -33,14 +33,13 @@ const YouCanHelp = ({ isTesting }) => {
...
-            xs={8}
+            xs={20}
             sm={20}
-            md={24}
-            xl={12}
+            md={10}
+            xl={10}
             src={HelpImage}
             alt="help"
           />
```
Comment (cuygur) [inline src/components/YouCanHelp/index.jsx:45]: New line need before export at line 50

**Review by cuygur — CHANGES_REQUESTED** [review]
in HD size(1280x800) I see like this. Hiding text can be solution. We need to discuss 

Screenshot 2020-11-22 023602


Please check below image;
image

**Review by halitbatur — APPROVED** [review]

**Review by bertacsevercan — APPROVED** [review]

**Comment by cuygur** [comment]
Very strange. I don't see like that. It's very important, Ihab will demo and his screen is HD. @ihab-sensei how do you see ?

**Comment by cuygur** [comment]
We solved the problem. Now it's perfect. Good job!

**Review by cuygur — APPROVED** [review]
Well done

**Pull request merged**